### PR TITLE
Feat/eae 7 generic form input

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: backend/main.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        }
+
+
+class SourceNameEnum(str, Enum):
+    hydro_merit_1k = "hydro_merit_1k"
+    hydro_merit = "hydro_merit"
+
+
+class HydroMT(BaseModel):
+    # Faroe Island
```
Comment (SiggyF) [inline backend/main.py:28]: What is this doing here?

File: backend/main.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    bbox: BBox
+    strord: Optional[PositiveInt]
+    source_name: SourceNameEnum = SourceNameEnum.hydro_merit_1k
+
+    class Config:
```
Comment (SiggyF) [inline backend/main.py:33]: this syntax .... (a class in a class). Can we not stick to a declarative approach (swagger/openapi3)?

File: src/components/MapComponent.vue [hunk 3]
```diff
@@ -40,33 +43,74 @@ export default {
...
+      })
     },
     drawFunction (e) {
+      // First delete all previous elements in the draw component, so we always have 1 bbox selected
+      // Store a new bbox and send the coordinates to the store
       this.draw.deleteAll()
       this.draw.add(e.features[0])
-      const NW = this.map.project(e.features[0].geometry.coordinates[0][3])
-      const SE = this.map.project(e.features[0].geometry.coordinates[0][1])
+      const North = Math.min(...e.features[0].geometry.coordinates[0][3])
```
Comment (SiggyF) [inline src/components/MapComponent.vue]: `N`? for consistency?

File: src/views/Editor.vue [hunk 4]
```diff
@@ -1,15 +1,123 @@
...
+
+      fetch(`${process.env.VUE_APP_EDITOR_SERVER}/hydromt`, {
+        method: 'POST',
+        body: JSON.stringify(jsonBody)
+      })
+        .then(response => {
+          return response.json()
+        })
+        .then(res => {
+          console.log(res)
```
Comment (SiggyF) [inline src/views/Editor.vue]: remove log

**Review by SiggyF — APPROVED** [review]
Frontend looks good. Remove the backend?

File: src/components/MapComponent.vue (hunk 3 above)
Comment (CindyvdVries) [reply src/components/MapComponent.vue]: check

File: src/views/Editor.vue (hunk 4 above)
Comment (CindyvdVries) [reply src/views/Editor.vue]: check

**Pull request merged**